src/pages/MeetingPage/MeetingPage.tsx

```diff
@@ -153,7 +158,7 @@ const MeetingPage = () => {
           <CheckboxGroup colorScheme="primary">
             <Stack>
               {categoryCheckboxes.map(checkbox => (
-                <Checkbox name="field" value={checkbox.value}>
+                <Checkbox name="field" value={checkbox.value} key={v4()}>
```

Comment:
It's not recommended to use a new uuid as the key , using a key attribute helps React identify which items have changed,the key should be a unique and stable identifier , but v4() will generate new value when component render

we can use `checkbox.title` for the key instead `v4()`

Reply:
Yes, you are right, I'm sorry, it was my oversight and I made a very basic mistake.

But I think it is better to have an id for the original data than `checkbox.title`, and it should be a habit to develop, so I added a unique and unchanging id to the original data in the database.

Then I changed `checkbox.title` to `checkbox.id`.